mapeditor/inspector/towneventswidget.cpp

```diff
@@ -179,15 +179,16 @@ void TownEventsDelegate::setModelData(QWidget * editor, QAbstractItemModel * mod
 
 void TownEventsDelegate::updateModelData(QAbstractItemModel * model, const QModelIndex & index) const
 {
-	QStringList eventLists;
+	QStringList eventList;
+	eventList += QObject::tr("Town Events:");
```

Comment:
```suggestion
	QStringList eventList = QObject::tr("Town Events:");
```

Reply:
Tried that already and it didn't work, but now I found `QStringList eventList(QObject::tr("Town Events:"))` ;)